Libraries/LibWeb/HTML/Parser/StackOfOpenElements.h

```diff
@@ -63,7 +63,12 @@ class StackOfOpenElements {
     void visit_edges(JS::Cell::Visitor&);
 
 private:
-    bool has_in_scope_impl(FlyString const& tag_name, Vector<FlyString> const&) const;
+    enum class CheckMathAndSVG : u8 {
+        No,
+        Yes,
+    };
+
+    bool has_in_scope_impl(FlyString const& tag_name, Vector<FlyString> const&, CheckMathAndSVG = CheckMathAndSVG::No) const;
```

Comment:
I'd rather not have the default `CheckMathAndSVG::No` value - it makes it clearer in the code what we're checking against.